### PR TITLE
Finish build variants

### PR DESCRIPTION
**Pull request by dragos-dobre**

### CODE REVIEW

File: app/src/main/java/org/openimis/imispolicies/AppInformation.java [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+import static org.openimis.imispolicies.BuildConfig.SHOW_CONTROL_NUMBER_MENU;
 
  public class AppInformation {
      public static class DomainInfo {
 
 //         URL address of openIMIS demo server => please update with your own Web Services base URL/IP
-         private static String _Domain = "https://demo.openimis.org/rest/";
+         private static String _Domain = API_BASE_URL;
 
          private static final String DEFAULT_RAR_PASSWORD = ")(#$1HsD";
```
Comment (delcroip) [inline app/src/main/java/org/openimis/imispolicies/AppInformation.java]: Could we add this one on the flavor config list ?

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -1,379 +1,378 @@
...
-  <string name="Quit">Arrêter de fumer</string>
-  <string name="Region">Région</string>
-  <string name="District">District</string>
-  <string name="Ward">Quartier</string>
-  <string name="Village">Village</string>
+  <string name="Feedback">Retour d\'information</string>
+  <string name="Sync">Synchroniser</string>
+  <string name="About">A propos</string>
+  <string name="Quit">Quitter</string>
+  <string name="Region">Promoteur</string>
```
Comment (delcroip) [inline app/src/main/res/values-fr/strings.xml]: I have to change that

File: app/src/main/java/org/openimis/imispolicies/AppInformation.java (hunk 1 above)
Comment (dragos-dobre) [reply app/src/main/java/org/openimis/imispolicies/AppInformation.java]: Good catch. I'll do this.

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (dragos-dobre) [reply app/src/main/res/values-fr/strings.xml]: done

**Pull request merged**